HOC/MotionWrapperHeadingTwo.tsx

```diff
@@ -12,14 +12,15 @@ const MotionWrapperHeadingTwo = ({
   delay,
   duration,
   additionalStyles,
+  once = true,
 }: MotionHeadingTwoProps) => {
   return (
     <motion.h2
       variants={fadeIn(direction, type, delay, duration)}
-      viewport={{ once: true, amount: 0.1 }}
+      viewport={{ once, amount: 0.1 }}
       initial="hidden"
       whileInView="show"
-      className={`${additionalStyles} text-[2.25rem] font-bold leading-[2.6rem] tracking-[-0.36px] md:text-[3rem] md:leading-[3.45rem] md:tracking-[-0.48px]`}
+      className={`${additionalStyles} `}
```

Comment:
```suggestion
      className={additionalStyles}
```
We don't need a template literal here